app/models/request.rb

```diff
@@ -7,6 +7,7 @@ class Request < ApplicationRecord
   validates :content, :user, :duedate, presence: true
 
   enum status: { pending: 0, accepted: 1, rejected: 2, commented: 3 }
+  enum open_closed: { open: 0, closed: 1 }
```

Comment:
i think, you can add that to the status enum.
`{ pending: 0, accepted: 1, rejected: 2, commented: 3, closed: 4 }`

0, 1, 2, and 3 are open.
4 is closed.